packages/ui-patterns/package.json

```diff
@@ -16,7 +16,10 @@
     "react-tooltip": "*",
     "reactflow": "*",
     "tsconfig": "*",
-    "ui": "*"
+    "ui": "*",
+    "cmdk": "^0.2.1",
+    "react-error-boundary": "^4.0.12",
+    "@supabase/auth-helpers-react": "*"
```

Comment:
We previously had this at `^0.4.2` - do we wanna use a wildcard here now?

Reply:
duno. i thought we use wildcards in here since they are installed in the apps

Reply:
ill use the numbered version